src/vs/workbench/contrib/preferences/electron-browser/settingsEditor2.ts

```diff
@@ -69,7 +70,7 @@ export class SettingsEditor2 extends BaseEditor {
 	private static SETTING_UPDATE_SLOW_DEBOUNCE: number = 1000;
 
 	private static readonly SUGGESTIONS: string[] = [
-		`@${MODIFIED_SETTING_TAG}`, '@tag:usesOnlineServices'
+		`@${MODIFIED_SETTING_TAG}`, '@tag:usesOnlineServices', `@${EXTENSION_SETTING_TAG}`
```

Comment:
If we don't want users to type in extension filters in the input, then we should not add it here.